src/cguittfont/CGUITTFont.h

```diff
@@ -258,6 +261,10 @@ namespace gui
 			virtual void draw(const core::stringw& text, const core::rect<s32>& position,
 				video::SColor color, bool hcenter=false, bool vcenter=false,
 				const core::rect<s32>* clip=0);
+			
+			virtual void draw(const EnrichedString& text, const core::rect<s32>& position,
+				video::SColor color, bool hcenter=false, bool vcenter=false,
+				const core::rect<s32>* clip=0);
```

Comment:
I could change that, but I wanted to stay consistent with the code that already existed (and the one in Irrlicht). Thus, should I change it?